analysis/study_definition.py

```diff
@@ -24,27 +24,36 @@
         "rate": "uniform",
         "incidence": 0.2,
     },
+        
+    # set the index date to be the campaign start -- change if necessasry
+    index_date = campaign_start, 
 
     # This line defines the study population
+    # change this to extract data for a different set of patients
     population=patients.satisfying(
         """
         registered
         AND
-        (age >= 18 AND age <= 110)
+        (age >= 80 AND age < 110)
+        AND
+        (sex = "M" OR sex = "F")
```

Comment:
My only query is that do we want to back to 80+ age range? Appreciate it is the most useful at this stage but we might forget to change in the future. Age restrictions could be applied via R or Stata. What do you think?

Reply:
have gone back to original and also removed carehome / prior covid exclusions